src/tabs/configuration.html

```diff
@@ -774,4 +774,15 @@
             </div>
         </div>
     </div>
+
+    <dialog class="dialogDynFiltersChange">
```

Comment:
This is in the wrong place here - it should really be in `main.html`, since it is used in two different tabs.